packages/cinterop/src/darwin/kotlin/io/realm/kotlin/internal/interop/RealmInterop.kt

```diff
@@ -1382,10 +1382,6 @@ actual object RealmInterop {
                         e.printStackTrace()
                     }
                 },
-                staticCFunction { userdata, asyncError ->
```

Comment:
How are these being reported now?

Reply:
Don't know why they were exposed in the C-API in the first place, because it sounds like it never should have been ... https://github.com/realm/realm-core/commit/2f1167e828e20e94fbff2f3d39dd609339f941e8